types/bootstrap/v5/modal.d.ts

```diff
@@ -0,0 +1,110 @@
+declare class Modal {
+    constructor(element: Element, options?: Partial<Modal.Options>);
+
+    /**
+     * Manually toggles a modal. Returns to the caller before the modal has
+     * actually been shown or hidden (i.e. before the shown.bs.modal or
+     * hidden.bs.modal event occurs).
+     */
+    toggle: () => void;
+
+    /**
+     * Manually opens a modal. Returns to the caller before the modal has
+     * actually been shown (i.e. before the shown.bs.modal event occurs).
+     */
+    show: () => void;
+
+    /**
+     * Manually hides a modal. Returns to the caller before the modal has
+     * actually been hidden (i.e. before the hidden.bs.modal event occurs).
+     */
+    hide: () => void;
+
+    /**
+     * Manually readjust the modal’s position if the height of a modal
+     * changes while it is open (i.e. in case a scrollbar appears).
+     */
+    handleUpdate: () => void;
+
+    /**
+     * Destroys an element's dropdown.
+     */
+    dispose: () => void;
+}
+
+declare namespace Modal {
+    enum Events {
+        /**
+         * This event fires immediately when the show instance method is called.
+         * If caused by a click, the clicked element is available as the
+         * relatedTarget property of the event.
+         */
+        show = 'show.bs.modal',
+
+        /**
+         * 	This event is fired when the modal has been made visible to the user
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/bootstrap/v5/modal.d.ts;
  Line: 45;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.